docs/README_RECON.md

```diff
@@ -0,0 +1,316 @@
+| Oracle      | number(10,5)  | trim(to_char(coalesce(<col_name>,0.0), ’99990.99999’))                 | cast(coalesce(<col_name>,0.0) as decimal(10,5)) | 1.00                    | 1.00000                 | this can be used for any precision and scale by adjusting accordingly in the transformation |
+| Snowflake   | array         | array_to_string(array_compact(<col_name>),’,’)                         | concat_ws(’,’, <col_name>)                      | [1,undefined,2]         | [1,2]                   | in case of removing "undefined" during migration(converts sparse array to dense array)      |
+| Snowflake   | array         | array_to_string(array_sort(array_compact(<col_name>), true, true),’,’) | concat_ws(’,’, <col_name>)                      | [2,undefined,1]         | [1,2]                   | in case of removing "undefined" during migration and want to sort the array                 |
+| Snowflake   | timestamp_ntz | date_part(epoch_second,<col_name>)                                     | unix_timestamp(<col_name>)                      | 2020-01-01 00:00:00.000 | 2020-01-01 00:00:00.000 | convert timestamp_ntz to epoch for getting a match between Snowflake and data bricks        |
+
+[[back to top](#remorph-reconciliation)]
+
+## Reconciliation Examples:
+
+For more Reconciliation Config examples, please refer to [sample_notebook][link].
+
+[link]: reconciliation-configs-examples.py
+
+[[back to top](#remorph-reconciliation)]
+
+# Frequently Asked Questions
+
+## Reconciler Questions
+
+<details>
+<summary>Can we reconcile for Databricks without UC as a target?</summary>
+
+***The reconciliation target is always Databricks with UC enabled. Reconciler supports non-uc Databricks only as a
+source.***
+</details>
+
+<details>
+<summary>What would happen if my dataset had duplicate records?</summary>
+
+***Duplicates are not handled in the reconciler. If run with duplicates, it would result in inconsistent output. We can
+implement
+some workarounds to handle the duplicates, and the solution varies from dataset to dataset.***
+</details>
+
+<details>
+<summary>Are User Transformations applicable for Schema Validations?</summary>
+
+***No. User Transformations are not applied for Schema Validation.Only select_columns,drop_columns and column_mapping is
+valid for schema validation.***
+</details>
+
+<details>
+<summary>Can we apply Aggregate or multi-column transformations as user transformations?</summary>
+
+***No. Aggregate transformations or multi-column transformations are not supported.***
+</details>
+
+<details>
+<summary>Does Reconciler support all complex data types?</summary>
+
+***Not all complex data types are supported currently.Reconciler do support UDFs for complex datatypes.Please refer here
+for examples.***
+</details>
+
+
+<details>
+<summary>Does Reconciler support `Threshold Validation` for report type as `row`?</summary>
```

Comment:
@ganeshdogiparthi-db  do we still skip threshold columns during `row` validation?